### DISCUSSION TITLE
Welcome to BCL Discussions

### DISCUSSION THREAD

**Discussion by brownbp1** [BCLCommons/bcl#1] [Announcements]
## Welcome
We’re using Discussions as a place to connect with other members of our community, publicly discuss ideas, and share knowledge related to the codebase. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.

To get started, comment below with an introduction of yourself and tell us about what you do (or want to do) with this community.

**Comment by jeffmendenhall** [comment]
Hi,
I'm Jeff, a long-time developer of the BCL. 
Happy to learn that we have forums now, and looking forward to supporting others on their BCL journey. 
For a while we practiced fight club rules about the BCL, except in publications, and user support has been largely through email. 
I lost count of how many times I answered questions about a particularly challenging application in the BCL; will be nice to have a central repo for some of that tribal knowledge.
Cheers
